<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

#frame, #frame_2
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}


QLabel
{
	color: rgb(107, 239, 0);
}

QPushButton, QToolButton
{
	background-color: rgb(73, 163, 0);
	color: rgb(0, 0, 0);
}

QComboBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
	selection-background-color: rgb(107, 239, 0);
    selection-color: rgb(0, 0, 0);
}

QComboBox QAbstractItemView
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QLineEdit
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QSpinBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QDoubleSpinBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QCheckBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

QCheckBox::indicator::checked
{
	background-color: rgb(107, 239, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QCheckBox::indicator::unchecked
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QMenu
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

QMenu::item:selected
{
	background-color: rgb(107, 239, 0);
	color: rgb(0, 0, 0);
}

QMenu::item:disabled
{
	background-color: rgb(0, 15, 0);
	color: rgb(27, 159, 0);;
}
</string>
  </property>
  <widget class="QLineEdit" name="lineEditRSBNFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>761</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Felis TU154 rsbn.dat</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>33</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXPlaneDirectory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>761</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>511</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X-Plane Folder (automagically searches for X-Plane files)</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>33</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxXP11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set if data is loaded from X-Plane 11</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>861</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X-Plane 10 settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAirwaysFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>761</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWaypointsFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>761</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>waypoints.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>airports.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ats.txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEarthnavFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>761</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAirportsFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>761</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNavaidsFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>761</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>earth_nav.dat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>navaids.txt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxManual">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manually set X-Plane files to load (only for X-Plane 10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>861</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="checkBoxAPTOverride">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Let data from custom installed airports override gateway and default airports</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X-Plane 11 settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>770</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDefaultLoadSaveDir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>761</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>761</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default folder for saving and loading flightplans aswell as saving PDF:s.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>33</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
